Add 11.23 release notes (#376)
mondoohq/docs: releases/2024-09-24-mondoo-11.23-is-out.md

---
slug: mondoo-11.23-is-out/
title: Mondoo 11.23 is out!
description: Announcing the 11.23 release of Mondoo with automatic drift detection, GitHub and GitLab ticketing support, improved asset tables, and more!
authors: [tim, chip]
image: /img/releases/2024-09-24-mondoo-11.23-is-out/policy_asset_table.png
tags: [release, mondoo]
---

## 🥳 Mondoo 11.23 is out! This release includes automatic drift detection, GitHub and GitLab ticketing support, improved asset tables, and more!

Get this release: [Installation Docs](https://mondoo.com/docs/cnspec/) | [Package Downloads](https://releases.mondoo.com/cnspec/) | [Docker Container](https://hub.docker.com/r/mondoo/cnspec)

---

## 🎉 NEW FEATURES

> ### GitHub, GitLab, and email ticketing
> 
> Mondoo users can now collect security findings into cases and assign those cases to external ticket systems for efficient tracking and resolution. Mondoo now additionally supports creating tickets in GitHub Issues and GitLab Issues, as well as sending ticket notifications via email. These options join our existing support for ticketing with Jira & Zendesk.
> 
> ### Create Cases automatically on asset drift
> 
> When scans go from passing to failing, you want to know quickly. Mondoo now automatically creates work items to investigate checks and vulnerability scans that have gone from previously passing to failing, and assigns that work via ticketing systems, so teams can respond rapidly to suspicious changes on any asset.
> 
> 

Reviewer: How about this?

```suggestion
### Added features in cases

This release further expands the capabilities of _cases_, Mondoo's feature for tracking remediation work. Cases let you turn security findings into tasks that automatically export to your existing project management or ticket system. That way, you can track and resolve Mondoo findings in your regular workflow.

#### New support for GitHub Issues, GitLab Issues, and email ticketing

Mondoo now supports creating tickets in GitHub Issues and GitLab Issues, and can also send ticket notifications via email. These options join our existing support for ticketing with Jira and Zendesk.

#### Create cases automatically on asset drift

When scans go from passing to failing, you want to know quickly. Now when an asset fails a check or vulnerability scan that it previously passed, Mondoo creates a case and shares it with your ticket system so you can respond rapidly. Mondoo can even group similar drift findings in a single case.

```

### New CIS Ubuntu 24.04 benchmark policy

Secure your Ubuntu 24.04 servers and containers with the brand new CIS Ubuntu 24.04 level 1 & 2 benchmarks. These policies include 295 total checks to secure your critical Ubuntu systems.

## 🧹 IMPROVEMENTS

### Updated CIS Azure Foundations 3.0 benchmark policy

> Evaluate your Azure subscriptions against the latest CIS recommendations with the updated CIS Azure Foundations benchmark version 3.0. This updated policy includes 15 all new checks as well as 69 updated checks.

Reviewer: ```suggestion
Evaluate your Azure subscriptions against the latest CIS recommendations with the updated CIS Azure Foundations benchmark version 3.0. This updated policy includes 15 all-new checks as well as 69 updated checks.
```

### Improved asset view in policies

> Out with the old and in with the new. Now you can view risk scores, risk factors, and last updated times for assets directly from the policy pages.

Reviewer: ```suggestion
Out with the old and in with the new: Now you can view risk scores, risk factors, and last updated times for assets directly from the policy pages.
```

![Policy Assets Table](/img/releases/2024-09-24-mondoo-11.23-is-out/policy_asset_table.png)

### Resource updates

#### github.repository

- New `codeOfConductFile` field
- New `supportFile` field
- New `securityFile` field

#### github.file

- New `exists` field

#### microsoft.conditionalAccess

- New resource with `namedLocations` field

#### microsoft.conditionalAccess.ipNamedLocation

- New resource with `name` and `trusted` fields

#### azure.subscription.networkservice.applicationGateway

- New `wafConfiguration` field using the new `azure.subscription.networkService.wafConfig` resource

#### azure.subscription.webService.appsite

- New `functions` field using the new `azure.subscription.webService.function` resource

## 🐛 BUG FIXES AND UPDATES

- Improve the performance of EC2 instance discovery.
- Fix tag filtering in the AWS provider.
- Avoid errors in the AWS Elasticsearch, SageMaker, and SNS resources when nil values are returned.
- Ensure that the AWS KMS resource returns all keys.
- Improve performance of data exports.
- Improve performance loading vulnerability data.

> - Show risk scores for CVE each CVE displayed on advisory pages.

Reviewer: ```suggestion
- Show risk scores for each CVE displayed on advisory pages.
```
